src/fonts/selection.json

```diff
@@ -0,0 +1,261 @@
+				"ligatures": "bubble2, comment2",
+				"name": "bubble",
+				"id": 110,
+				"order": 64,
+				"prevSize": 32,
+				"code": 59758
+			},
+			"setIdx": 1,
+			"setId": 0,
+			"iconIdx": 110
+		},
+		{
+			"icon": {
+				"paths": [
+					"M853.342 0h-682.656c-93.874 0-170.686 76.81-170.686 170.69v682.622c0 93.938 76.812 170.688 170.686 170.688h682.656c93.876 0 170.658-76.75 170.658-170.69v-682.62c0-93.88-76.782-170.69-170.658-170.69zM853.342 128c7.988 0 15.546 2.334 22.020 6.342l-363.362 300.404-363.354-300.4c6.478-4.010 14.044-6.346 22.040-6.346h682.656zM170.686 896c-1.924 0-3.82-0.146-5.684-0.408l225.626-312.966-29.256-29.254-233.372 233.37v-611.138l384 464.396 384-464.394v611.136l-233.372-233.37-29.254 29.254 225.628 312.968c-1.858 0.26-3.746 0.406-5.662 0.406h-682.654z"
+				],
+				"attrs": [],
+				"isMulticolor": false,
+				"isMulticolor2": false,
+				"tags": [
+					"mail",
+					"contact",
+					"support",
+					"newsletter",
+					"letter",
+					"email",
+					"envelop",
+					"social"
+				],
+				"defaultCode": 60036,
+				"grid": 16
+			},
+			"attrs": [],
+			"properties": {
+				"ligatures": "mail3, contact3",
+				"name": "mail",
+				"id": 388,
+				"order": 66,
+				"prevSize": 32,
+				"code": 60036
+			},
+			"setIdx": 1,
+			"setId": 0,
+			"iconIdx": 388
+		},
+		{
+			"icon": {
+				"paths": [
+					"M1024 226.4c-37.6 16.8-78.2 28-120.6 33 43.4-26 76.6-67.2 92.4-116.2-40.6 24-85.6 41.6-133.4 51-38.4-40.8-93-66.2-153.4-66.2-116 0-210 94-210 210 0 16.4 1.8 32.4 5.4 47.8-174.6-8.8-329.4-92.4-433-219.6-18 31-28.4 67.2-28.4 105.6 0 72.8 37 137.2 93.4 174.8-34.4-1-66.8-10.6-95.2-26.2 0 0.8 0 1.8 0 2.6 0 101.8 72.4 186.8 168.6 206-17.6 4.8-36.2 7.4-55.4 7.4-13.6 0-26.6-1.4-39.6-3.8 26.8 83.4 104.4 144.2 196.2 146-72 56.4-162.4 90-261 90-17 0-33.6-1-50.2-3 93.2 59.8 203.6 94.4 322.2 94.4 386.4 0 597.8-320.2 597.8-597.8 0-9.2-0.2-18.2-0.6-27.2 41-29.4 76.6-66.4 104.8-108.6z"
+				],
+				"attrs": [],
+				"isMulticolor": false,
+				"isMulticolor2": false,
+				"tags": ["twitter", "brand", "tweet", "social"],
+				"defaultCode": 60054,
+				"grid": 16
+			},
+			"attrs": [],
+			"properties": {
+				"ligatures": "twitter, brand16",
+				"name": "twitter",
+				"id": 406,
+				"order": 67,
+				"prevSize": 32,
+				"code": 60054
+			},
+			"setIdx": 1,
+			"setId": 0,
+			"iconIdx": 406
+		}
+	],
+	"height": 1024,
+	"metadata": {
+		"name": "icomoon"
+	},
+	"preferences": {
+		"showGlyphs": true,
+		"showCodes": true,
+		"showQuickUse": true,
+		"showQuickUse2": true,
+		"showSVGs": true,
+		"fontPref": {
+			"prefix": "icon-",
+			"metadata": {
+				"fontFamily": "icomoon"
+			},
+			"metrics": {
+				"emSize": 1024,
+				"baseline": 6.25,
+				"whitespace": 50
+			},
+			"embed": false
+		},
+		"imagePref": {
+			"prefix": "icon-",
+			"png": true,
+			"useClassSelector": true,
+			"color": 0,
+			"bgColor": 16777215,
+			"name": "icomoon",
+			"classSelector": ".icon"
+		},
+		"historySize": 50,
+		"gridSize": 16
+	}
+}
```

Comment:
I think we already had some icon font (in Material UI?). Can't we just keep that one so we have less to maintain? (see https://material-ui.com/api/icon/). I prefer sticking to this `Icon` component so we don't need an extra dependency.

Reply:
removed